apps/backend/src/libs/modules/config/base-config.module.ts

```diff
@@ -87,6 +87,14 @@ class BaseConfig implements Config {
 					format: Number,
 				},
 			},
+			MAPBOX_DIRECTIONS_API: {
```

Comment:
let's just have everything related to mapbox under one group
```suggestion
			MAPBOX: {
```